<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="8,2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="board_players_layout" stretch="1,12,1,3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="board_layout" rowstretch="5,5,5,5,5,5,5,5,5,5,5" columnstretch="5,5,5,5,5,5,5,5,5,5,5" rowminimumheight="0,0,0,0,0,0,0,0,0,0,0" columnminimumwidth="0,0,0,0,0,0,0,0,0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="8">
         <widget class="QWidget" name="tile_9" native="true"/>
        </item>
        <item row="1" column="10">
         <widget class="QWidget" name="tile_12" native="true"/>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="tile_2" native="true"/>
        </item>
        <item row="0" column="10">
         <widget class="QWidget" name="tile_11" native="true"/>
        </item>
        <item row="0" column="4">
         <widget class="QWidget" name="tile_5" native="true"/>
        </item>
        <item row="7" column="10">
         <widget class="QWidget" name="tile_18" native="true"/>
        </item>
        <item row="0" column="2">
         <widget class="QWidget" name="tile_3" native="true"/>
        </item>
        <item row="4" column="10">
         <widget class="QWidget" name="tile_15" native="true"/>
        </item>
        <item row="0" column="9">
         <widget class="QWidget" name="tile_10" native="true"/>
        </item>
        <item row="0" column="5">
         <widget class="QWidget" name="tile_6" native="true"/>
        </item>
        <item row="8" column="10">
         <widget class="QWidget" name="tile_19" native="true"/>
        </item>
        <item row="2" column="10">
         <widget class="QWidget" name="tile_13" native="true"/>
        </item>
        <item row="6" column="10">
         <widget class="QWidget" name="tile_17" native="true"/>
        </item>
        <item row="0" column="6">
         <widget class="QWidget" name="tile_7" native="true"/>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="tile_1" native="true"/>
        </item>
        <item row="3" column="10">
         <widget class="QWidget" name="tile_14" native="true"/>
        </item>
        <item row="0" column="7">
         <widget class="QWidget" name="tile_8" native="true"/>
        </item>
        <item row="0" column="3">
         <widget class="QWidget" name="tile_4" native="true"/>
        </item>
        <item row="9" column="10">
         <widget class="QWidget" name="tile_20" native="true"/>
        </item>
        <item row="5" column="10">
         <widget class="QWidget" name="tile_16" native="true"/>
        </item>
        <item row="10" column="10">
         <widget class="QWidget" name="tile_21" native="true"/>
        </item>
        <item row="10" column="9">
         <widget class="QWidget" name="tile_22" native="true"/>
        </item>
        <item row="10" column="8">
         <widget class="QWidget" name="tile_23" native="true"/>
        </item>
        <item row="10" column="7">
         <widget class="QWidget" name="tile_24" native="true"/>
        </item>
        <item row="10" column="6">
         <widget class="QWidget" name="tile_25" native="true"/>
        </item>
        <item row="10" column="5">
         <widget class="QWidget" name="tile_26" native="true"/>
        </item>
        <item row="10" column="4">
         <widget class="QWidget" name="tile_27" native="true"/>
        </item>
        <item row="10" column="3">
         <widget class="QWidget" name="tile_28" native="true"/>
        </item>
        <item row="10" column="2">
         <widget class="QWidget" name="tile_29" native="true"/>
        </item>
        <item row="10" column="1">
         <widget class="QWidget" name="tile_30" native="true"/>
        </item>
        <item row="10" column="0">
         <widget class="QWidget" name="tile_31" native="true"/>
        </item>
        <item row="9" column="0">
         <widget class="QWidget" name="tile_32" native="true"/>
        </item>
        <item row="8" column="0">
         <widget class="QWidget" name="tile_33" native="true"/>
        </item>
        <item row="7" column="0">
         <widget class="QWidget" name="tile_34" native="true"/>
        </item>
        <item row="6" column="0">
         <widget class="QWidget" name="tile_35" native="true"/>
        </item>
        <item row="5" column="0">
         <widget class="QWidget" name="tile_36" native="true"/>
        </item>
        <item row="4" column="0">
         <widget class="QWidget" name="tile_37" native="true"/>
        </item>
        <item row="3" column="0">
         <widget class="QWidget" name="tile_38" native="true"/>
        </item>
        <item row="2" column="0">
         <widget class="QWidget" name="tile_39" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="tile_40" native="true"/>
        </item>
        <item row="2" column="2" rowspan="7" colspan="7">
         <widget class="QLabel" name="image">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/images/res/monopoly_logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,20">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,3,2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Money</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Token</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="players_widget_layout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Current Player</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentPlayer">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="action_layout" stretch="1,12,1,3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>MessageBoard</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="message_board">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="rollDice">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Roll Dice</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="endTurn">
          <property name="text">
           <string>End Turn</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="buyProperty">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Buy Property</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="jailBail">
          <property name="text">
           <string>Pay Jail Bail</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
